### ISSUE TITLE
Support the .NET core.

### ISSUE THREAD

**Issue by PJB3005** [space-wizards/RobustToolbox#174]
Problem is that SFML is probably completely unable to run the Core so...

**Comment by elwray** [comment]
As I known SDL2 has .NET core bindings. But migrate from SFML to SDL2 is complex and unnecessary task for this moment.

**Comment by PJB3005** [comment]
SDL2 has pretty awful C# bindings though.

**Comment by ZoldorfTheWizard** [comment]
R e m o v e   sfml #251

**Comment by gbasood** [comment]
You know, when we have issues like this, it'd be useful to actually describe *why* we did/will do a thing.

**Comment by PJB3005** [comment]
You know since we're using Godot on the client anyways, supporting .NET Core for the server only might make sense.

**Comment by ZoldorfTheWizard** [comment]
+1 for supporting .NET core

**Comment by Acruid** [comment]
At the moment .net Core does not support appdomains or assembly unloading. Unless this changes, the only real option is to load addons into the same domain as all the content/assembly code. The only defense against malicious addons will be Namespace whitelisting with Cecil, and any OS privilege restrictions to the game process.

**Comment by Cyberboss** [comment]
as long as your plugins don't have dependencies, you can use a custom AssemblyLoadContext which is pmuch an AppDomain without the remoting bullshit

**Comment by PJB3005** [comment]
I messed around with this a bit.

I got the server loading but didn't yet bother to make the client load. It seems pretty feasible to move to .NET Core and .NET Standard.

My idea is:
* Shared and client become .NET Standard
* Server becomes .NET Core.

I'm not sure what to do about the client, because I'd love for `Content.Client` to use .NET Standard no matter what, but that's not possible if `SS14.Client` is a .NET Core project.
